src/app/common/components/flows-table/flows-table-processing-base.component.ts

```diff
@@ -88,7 +87,7 @@ export abstract class FlowsTableProcessingBaseComponent extends BaseComponent {
                 this.currentPage,
                 status,
                 filterInitiatorOptions,
-                datasets && datasets.length ? datasets.map((item) => item.id) : [],
+                datasets && datasets.length ? (datasets as Dataset[]).map((item) => item.id) : [],
```

Comment:
Cast like this (here and 2 more places), are not required. Everything compiles and lints perfectly.

Reply:
Fixed